### PR TITLE
feat: Update Phase 1 Weekly Training Schedule

### PR DESCRIPTION
**Pull request by gtempus**
I acknowledge that the CLI project comes later than usual.
My idea is to have several smaller projects per week or so that the
apprentices can use them to focus on the current concepts and continue to
practice what they've learned so far.

### CODE REVIEW

File: phases/1-programming.md [hunk 1]
```diff
@@ -7,8 +7,31 @@ Read these books and add a 100 - 200 word review to your weekly journal. This sh
 
 * [Getting Real][], 37signals
 * [Passionate Programmer][], Chad Fowler
+* [Pragmatic Programmer][],  Andrew Hunt & Dave Thomas
 
 #### **Training:**
+* Week 0: CLI & Command Line
+* Week 1: Git at Sparkbox
+  * ♫ "Shunting trucks and hauling freight" ♫
```
Comment (cromwellryan) [inline phases/1-programming.md]: Lol 🚂

File: phases/1-programming.md [hunk 2]
```diff
@@ -7,8 +7,31 @@ Read these books and add a 100 - 200 word review to your weekly journal. This sh
...
+* Week 2: Functional Programming
+  * Breaking a Problem Down by Breaking a Problem Down
+  * Reason Your Way to Fame and Fortune
+* Week 3: Object Oriented Programming
+  * "Mr. Watson, come here. I want to see you." --A. Bell
+  * Cells: The Building Blocks of Life!
```
Comment (cromwellryan) [inline phases/1-programming.md]: Conway’s?

File: phases/1-programming.md [hunk 3]
```diff
@@ -7,8 +7,31 @@ Read these books and add a 100 - 200 word review to your weekly journal. This sh
...
+* Week 4: SOLID Principles
+  * You Can't Make All the Priciples Happy All of the Time
```
Comment (cromwellryan) [inline phases/1-programming.md]: _Principles_ spelling

File: phases/1-programming.md [hunk 4]
```diff
@@ -7,8 +7,31 @@ Read these books and add a 100 - 200 word review to your weekly journal. This sh
...
+  * Use the Force
+* Week 5: Testing
+  * "Weeks of programming can save you hours of planning"
```
Comment (cromwellryan) [inline phases/1-programming.md]: I’m loving the quotes 😂

File: phases/1-programming.md [hunk 5]
```diff
@@ -7,8 +7,31 @@ Read these books and add a 100 - 200 word review to your weekly journal. This sh
...
+  * Test After: Show Correctness
+  * Test Before: Show Intention
+  * Test to Drive Design: Show Up On Time While Being Intentionally Correct
+* Week 6: Introduce CLI Project
+* Week 7: Design Patterns
+  * You Mean There's a Name for What I Did?
+* Week 8: Evolving Your Design
+  * Damnit Jim, I'm a programmer not a philosopher...oh, wait
```
Comment (catheraaine) [inline phases/1-programming.md]: technically spelled "dammit"

File: phases/1-programming.md [hunk 6]
```diff
@@ -46,6 +69,7 @@ Read these books and add a 100 - 200 word review to your weekly journal. This sh
 
 [Getting Real]: http://gettingreal.37signals.com/
 [Passionate Programmer]: http://www.amazon.com/The-Passionate-Programmer-Remarkable-Development/dp/1934356344
+[Pragmatic Programmer]: https://www.amazon.com/Pragmatic-Programmer-Journeyman-Master/dp/020161622X
```
Comment (catheraaine) [inline phases/1-programming.md]: do we own a copy of this one, @cromwellryan ? We might need to get one or two.

Comment (cromwellryan) [reply phases/1-programming.md]: Seems we don't. Let's get a digital copy. Leah can help with that.

File: phases/1-programming.md (hunk 2 above)
Comment (gtempus) [reply phases/1-programming.md]: Yes, and reinforcing data hiding and encapsulation. I want to look at how message passing works in biology.

**Pull request merged**